presentation/courses/addCourse.php

```diff
@@ -50,51 +51,22 @@
 <h2>Add Course</h2>
 <form method="post" action="../../data/courses/addCourseScript.php">
     <div>
-        <p>Course ID
-            <input name="course_code" type="text" size="10" required/><br/></p>
-        <p>Course Title
-            <input name="title" type="text" size="40" required/>
-        </p><br/>
+        <p>Student ID
+            <label>
+                <input name="id" type="number" size="10" placeholder="Student ID" required/>
+            </label><br/></p>
+        <p>Course Code
+            <label>
+                <input name="course_code" type="text" size="10" placeholder="Course Code" required/>
```

Comment:
Course code should maybe be a dropdown or at least an autofill.

Reply:
Optional features for later, but good idea.